src/global-styles.scss

```diff
@@ -1,14 +1,71 @@
-$example-color: cornflowerblue;
+// gray-700
+// $bg-color: #374151;
 
-.example-class {
-    color: $example-color;
+// zinc-900
+$bg-color: #18181b;
+$text-color: #f8fafc;
+
+// gray-800
+// $card-lv-1-color: #1f2937;
+
+// zinc-700
+$card-lv-1-color: #3f3f46;
+
+
+
+html {
+    background-color: $bg-color;
+    color: $text-color;
+    height: 100%;
+}
+
+body {
+    height: 100%;
+}
+
+.sveltekit-container {
```

Comment:
Alternative:
```css
body {
     display: flex;
     flex-flow: column;
}
```
Flex is your best friend for reactive positioning.  It's so important, there's a frog game about it https://flexboxfroggy.com/.


Regardless, if you do explicitly want to style that div, prefer styling it by id (#sveltekit-container) instead of by class.  Classes imply reuse, and I don't think we'll be reusing that class anywhere.

Reply:
Actually, neither piece of css code is doing anything.

Reply:
I had to give all the things `height:100%` or the app would be squished.

Reply:
I agree with changing to id.  I am a fan of flexbox, but I don't see the place where flex would come in handy here.